pkg/cli/cli.go

```diff
@@ -166,8 +172,8 @@ func NewDefaultOcCommand(in io.Reader, out, errout io.Writer) *cobra.Command {
 	return cmd
 }
 
-func NewOcCommand(in io.Reader, out, err io.Writer) *cobra.Command {
-	warningHandler := rest.NewWarningWriter(err, rest.WarningWriterOptions{Deduplicate: true, Color: kterm.AllowsColorOutput(err)})
+func NewOcCommand(o kubecmd.KubectlOptions) *cobra.Command {
```

Comment:
This is breaking interface change. Thereby, I searched external usage of this function and there is none found which means that this change is safe.